pkg/interop/convert/convert.go

```diff
@@ -3,15 +3,15 @@ package convert
 
 // ToInteger converts it's argument to an Integer.
 func ToInteger(v interface{}) int64 {
-	return 0
+	return v.(int64)
```

Comment:
We use `int64` here and yet we use `int` for `Pow` and `Sqrt`. I think we should use one type everywhere and it's better to use the widest one available.

Reply:
I think `int` is better because it is default (no need for constant conversions) and is somewhat expected to have platform-dependent bit-width (256 in our case).

Reply:
Well, Go compiler wouldn't allow you to put 256-bit constant into `int` and 64 bits is the maximum integer width supported by Go proper (`int` is either 32- or 64-bit, it can't be 256-bit per spec, it'd be easy to have proper native 256-bit integer in C, but not in Go).

Other SDKs usually provide some `BigInteger` types, but I don't see any use case for it at the moment (so that it couldn't be done with regular int types).

Let's settle on `int` for now and then we'll see.